src/main/java/com/softserve/academy/spaced/repetition/service/CardService.java

```diff
@@ -10,21 +11,46 @@
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
+import java.util.ArrayList;
+import java.util.Date;
 import java.util.List;
 
 @Service
 public class CardService {
 
     private final static int CARDS_NUMBER = 10;
 
-    @Autowired
-    private CardRepository cardRepository;
+    private final CardRepository cardRepository;
 
-    @Autowired
-    private DeckRepository deckRepository;
+    private final DeckRepository deckRepository;
+
+    private final UserService userService;
 
     @Autowired
-    private UserService userService;
+    public CardService(CardRepository cardRepository, DeckRepository deckRepository, UserService userService) {
+        this.cardRepository = cardRepository;
+        this.deckRepository = deckRepository;
+        this.userService = userService;
+    }
+
+    public List<Card> getLearningCards(Long deckId) throws NotAuthorisedUserException {
+        User user = userService.getAuthorizedUser();
+        String email = user.getAccount().getEmail();
+        List<Card> learningCards = new ArrayList<>();
+        if (user.getAccount().getLearningRegime() == LearningRegime.STATUS_LEARNING.ordinal()) {
```

Comment:
getLearningRegime() should return LearningRegime enum.

You should compare two enums, not integers.